### ISSUE TITLE
what triggers the deletion of history record?

### ISSUE THREAD

**Issue by tenge-amzn** [smrq/chrome-history-blacklist#3]
Sometimes it just doesn't delete the history of listed domains..

**Comment by NA0341** [comment]
I don't think that it deletes history entries - but rather prevents them from being created in the first place.
As it says: _Any URL that matches any entry will be prevented from entering your browser history._

As to why the extension doesn't work as expected: provide the following info so the dev can help you:

+ Your OS & Version
+ Your Browser & Version
+ Any unusual things or customizations about your system or browser

I'm using Vivaldi on Manjaro Linux and it works perfectly fine for me.
I have currently 70 filter entries.
